frontend/src/components/molecules/ContractLink.vue

```diff
@@ -30,8 +30,26 @@ import Tooltip from '~/components/atoms/Tooltip.vue'
 
 type Props = {
 	address?: string | null
+	contractAddressIndex?: number | null
+	contractAddressSubIndex?: number | null
 	iconSize?: string
 	hideTooltip?: boolean
 }
 const props = defineProps<Props>()
+const drawer = useDrawer()
+const emit = defineEmits(['blur'])
+const emitBlur = (newTarget: FocusEvent) => {
+	emit('blur', newTarget)
+}
+
+const handleOnClick = () => {
+	props.contractAddressIndex &&
```

Comment:
If `contractAddressIndex` is `0`, then this check fails. You should check in the same way as for the `subindex`. I don't recall if `0` is actually allowed for an `index`, but that would be implicit knowledge.